src/test/java/weizberg/citibike/StationMethodsTest.java

```diff
@@ -0,0 +1,58 @@
+package weizberg.citibike;
+
+
+import org.junit.jupiter.api.Test;
+import weizberg.citibike.json.Station;
+
+
+import java.util.Map;
+
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+public class StationMethodsTest {
+
+
+
+
+    @Test
+    public void stationStatus() {
+        //given
+        MergeStationData mergeStationData = new MergeStationData();
+        Map<String, Station> stationMap = mergeStationData.getStationsMap();
+
+
+        //when
+        StationMethods stationMethods = new StationMethods();
```

Comment:
The way you have it now, you are creating two MergeStationData objects. 
One is inside StationMethods and one is on line 22.